OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -244,6 +244,12 @@ public interface IRenderModifier
 		IEnumerable<Rectangle> ModifyScreenBounds(Actor self, WorldRenderer wr, IEnumerable<Rectangle> r);
 	}
 
+	[RequireExplicitImplementation]
+	public interface ITilesetSpecificPaletteInfo : ITraitInfoInterface
```

Comment:
Perhaps

```suggestion
	public interface IPaletteWithTilesetInfo : ITraitInfoInterface
```

Reply:
or something like that. I'm not a fan of the current name

Reply:
We already have `TilesetSpecificSpriteSequence`, which is where I got the idea. "Palette with tileset info" is good, but since this is an info type it would be `IPaletteWithTilesetInfoInfo`, otherwise it's just "palette with tileset". 😅

Reply:
No, it would be `PaletteInfoWithTileset`